### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by smburdick**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # tqec
 
-Design automation software tools for Topological Quantum Error Correction
+Design automation software tools (frontend and backend) for Topological Quantum Error Correction.
```
Comment (vtomole) [inline README.md:3]: Frontend and backend delineations are not needed here. This sentence just says what `TQEC` is. Let's add a section in the `README` called `Architecture` where we give a gist of the frontend and backend.

Comment (smburdick) [reply README.md:3]: I agree. I can add some basic details, but I think a deeper dive is warranted for a separate wiki page.

**Review by nelimee — COMMENTED** [review]
Having the frontend files in a subfolder and the backend in the main folder seems weird to me. 
In my opinion, the installation guides should either be:
- both in the main `README.md` file
- both in a separate `how_to_install.md` file that is linked in the main `README.md` file.

In the future, I even think that the installation process should be "how to install the TQEC stack fully". Installing only one of the two components (frontend or backend) is a development process, and can be documented elsewhere.

**Comment by smburdick** [comment]
I'm generally biased towards discrete wiki pages that explains a particular thing in detail.

**Comment by nelimee** [comment]
> I'm generally biased towards discrete wiki pages that explains a particular thing in detail.

In the short term, I agree with Wiki pages. In the long term, #109 will likely fulfill that purpose.

**Comment by smburdick** [comment]
> Having the frontend files in a subfolder and the backend in the main folder seems weird to me. In my opinion, the installation guides should either be:
> 
> * both in the main `README.md` file
> * both in a separate `how_to_install.md` file that is linked in the main `README.md` file.
> 
> In the future, I even think that the installation process should be "how to install the TQEC stack fully". Installing only one of the two components (frontend or backend) is a development process, and can be documented elsewhere.

Installation guides are conventionally placed towards the top of `README.md`

**Comment by smburdick** [comment]
Moving to a new branch: https://github.com/QCHackers/tqec/pull/158

**Pull request closed without merging**